packages/web/src/components/create-playlist/PlaylistForm.module.css

```diff
@@ -0,0 +1,70 @@
+.playlistForm {
+  display: flex;
+  align-items: stretch;
+}
+
+.playlistForm > .form {
+  flex: 1 auto;
+  margin-left: 24px;
+  display: flex;
+  flex-direction: column;
+}
+
+.playlistForm > .form > *:not(:last-child) {
```

Comment:
nit: Same here - could this be less specific?

Reply:
just a callout this was all there before, but yeah youre prob right!